src/core/ddsc/tests/whc.c

```diff
@@ -191,6 +211,23 @@ static void test_whc_end_state(dds_durability_kind_t d, dds_reliability_kind_t r
       ret = dds_write (writer, &sample_keyless);
       CU_ASSERT_FATAL (ret == DDS_RETCODE_OK);
     }
+
+    /* if history is truly keep all, there may never be more data present than the max of the
+       history depth(s) */
```

Comment:
I think this comment should be 'if history is truly keep _last_, ...'.

Reply:
Oops - fixing